### ISSUE TITLE
zero_ad missing 

### ISSUE THREAD

**Issue by viniciuscr** [brollb/simple-0ad-example#5]
Could you please include zero_ad, publish it on pip or at least put it on its own repository, so I don´t have to clone everything just to get it.

**Comment by brollb** [comment]
Good idea. I will publish this to PyPI when I get a couple free cycles!
